core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -124,7 +124,7 @@ public SetupWizard() {
      *
      * If you do not provide any value to that system property, the default admin account will not have an API Token.
      *
-     * @since TODO (for the existence of the sysprop, not the availability to plugin)
+     * @since 2.260 (with NoExternalUse)
```

Comment:
I rephrased this one because it seemed ambiguous/unclear to me as it was before.